app/components/pages/CustomFieldsSettings/tests/CreateEditCustomFieldContent.test.js

```
@@ -0,0 +1,187 @@
+        data-testid={dataTestId}
+        value={content}
+        onChange={onChange}
+        onKeyDown={handleKeyDownCallback}
+      />
+      {list?.map((item) => (
+        <div key={item.id} data-testid="enum-value-item">
+          {item.tag}
+          <button
+            onClick={() => handleRemoveTagCallback(item.id)}
+            data-testid="remove-enum-value-button"
+          >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)